src/modules/internet/index.ts

```diff
@@ -640,7 +640,7 @@ export class InternetModule extends ModuleBase {
     } = options;
 
     let result: string;
-    const strategy = this.faker.number.int(hasLastName ? 1 : 2);
+    const strategy = this.faker.number.int(hasLastName ? 1 : 2) as 0 | 1 | 2;
```

Comment:
Should we rewrite this section, to make the code more readable?
(Because I initially questioned myself, why is 0 a possible value, cause there is only 1 or 2. And my brain skipped the part that 1 or 2 is "the max" value, and so 0 is included as lower bound.)

```suggestion
    const strategy = this.faker.number.int({ max: hasLastName ? 1 : 2 }) as 0 | 1 | 2;
```

Reply:
I'd rather suggest to extract this logic into it's own function. That way we don't have to assert `as 0 | 1 | 2` multiple times. I'm generally more in favor to write explicit code. That would be inconsistent with the rest of the codebase, tho.

Reply:
😮 re-reading these lines now it clicked:
We should rewrite it to something like this:

```ts
const strategies = ['name-of-strat-1', 'name-of-strat-2']
if (hasLastname) strategies.push('name-of-strat-3')
const strategy = this.faker.helpers.arrayElement(strategies)
```

That would make it much more clear